interface/modules/zend_modules/public/js/installer/action.js

```diff
@@ -5,209 +5,233 @@
  * @link      https://www.open-emr.org
  * @author    Jacob T.Paul <jacob@zhservices.com>
  * @author    Vipin Kumar <vipink@zhservices.com>
+ * @author    Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2020 Jerry Padgett <sjpadgett@gmail.com>
  * @author    Remesh Babu S <remesh@zhservices.com>
  * @copyright Copyright (c) 2013 Z&H Consultancy Services Private Limited <sam@zhservices.com>
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
 
-function register(status,title,name,method,type){
-	$.post("./Installer/register", { st: status, mod_title: title, mod_name: name, mod_method:method,mtype:type},
-	   function(data) {
-			if(data == "Success") {
-				window.location.reload();
-      } else {
-        var resultTranslated = js_xl(data);
-				$('#err').html(resultTranslated.msg).fadeIn().delay(2000).fadeOut();
-      }
-	  }
-	);
+function register(status,title,name,method,type)
+{
+    $.post(
+        "./Installer/register",
+        { st: status, mod_title: title, mod_name: name, mod_method:method,mtype:type},
+        function (data) {
+            if (data == "Success") {
+                window.location.reload();
+            } else {
+                  var resultTranslated = js_xl(data);
+                $('#err').html(resultTranslated.msg).fadeIn().delay(2000).fadeOut();
+            }
+        }
+    );
 }
 
-function manage(id,action){
+function manage(id,action)
+{
+    if (action == 'unregister') {
+        if (!confirm("Please Comfirm with OK to Unregister this Module.")) {
```

Comment:
mispell in Comfirm

Reply:
Ya'all mean you'in's don't spell it that way on west coast?